lineman/app/components/attachment/attachment.coffee

```diff
@@ -0,0 +1,12 @@
+angular.module('loomioApp').directive 'attachment', ->
```

Comment:
Do you mind calling this attachmentPanel? it's best if BEM names have 2 words, so they're easy to search and replace.